packages/wallets/src/components/DerivAppsSection/DerivAppsSection.scss

```diff
@@ -23,3 +23,12 @@
         user-select: none;
     }
 }
+
+.wallets-success-modal {
```

Comment:
I'm wondering to which component it belongs?

Reply:
removed the styles from here and extracted out to a different component and scss file